Mods/src/PuppiMod.cc

```diff
@@ -38,53 +38,62 @@ PuppiMod::PuppiMod(const char *name, const char *title) :
   fApplyLowPUCorr(kTRUE),
   fUseEtaForAlgo(kTRUE),
   fEtaForAlgo(2.5),
-  fDumpingPuppi(kFALSE)
+  fDumpingPuppi(kFALSE),
+  fMapper(0)
 {
   // Constructor.
 }
 
 //--------------------------------------------------------------------------------------------------
-PuppiMod::~PuppiMod() {} 
```

Comment:
Be consistent with the coding style! If you decide that the function implementation comes a line after the function name (and that's the style used everywhere else in Bambu too), that is what you should do everywhere.